web/src/api/api.ts

```diff
@@ -170,6 +179,11 @@ function parseMetadata(data: d3.DSVRowArray): Metadata {
   );
 }
 
+export async function fetchBlocks(diffId: number, kmers: ObjMap<Kmer>): Promise<Array<Block>> {
```

Comment:
See my comment in another PR. I would:
- Keep the `blocks` field in `Diff`, but make it nullable.
- Use this method to populate a `Diff` with its blocks.